### PR TITLE
Chore/add ktlint

### PR DESCRIPTION
**Pull request by youngman3009**
Added the Pinterest recommended plugin for setting up Ktlint.
Disabled the very annoying import ordering rule, but can be changed if any major objections! Tend to do this due to discrepencies between Android Studio and Ktlint rules.
Enabled linting on CI.
Updated existing code to match Ktlint rules.

### CODE REVIEW

**Comment by youngman3009** [comment]
Ahhhhh why is the danger thing failing :scream:

**Comment by Chesire** [comment]
The ktlint import ordering should be fixed in the latest version (or one before), so I think its worth having that run again

**Comment by Chesire** [comment]
> Ahhhhh why is the danger thing failing 😱

It looks like when you run GitHub actions, it uses your environment variables not the target repos (which makes sense I guess, so you can't force a deploy), so it's struggling to run Danger so there is no Danger token use.

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[*.{kt,kts}]
+disabled_rules=import-ordering
```
Comment (Chesire) [inline .editorconfig:2]: Newer versions of ktlint seems to have fixed this for Android projects, it should now work without the need to disable this rule.

**Comment by Chesire** [comment]
Closes #3

File: .editorconfig (hunk 1 above)
Comment (youngman3009) [reply .editorconfig:2]: Oh perfect, did not know that, I'll remove this!

**Review by Chesire — APPROVED** [review]

**Review by ShaunPlummer — APPROVED** [review]
Changes look good to me

**Pull request merged**